Iceberg-TipUI/IceTipRepositoryBrowser.class.st

```diff
@@ -80,7 +80,7 @@ IceTipRepositoryBrowser >> initializeWidgets [
 	super initializeWidgets.	
 	
 	sidebarTree := self newIceOutline.
-	historyPanel := self instantiate: IceTipHistoryBrowser on: self model headModel.
+	historyPanel := self instantiate: IceTipHiedraHistoryBrowser on: self model headModel.
```

Comment:
@tinchodias Or we should make it conditional here to use IceTipHistoryBrowser or IceTipHiedraHistoryBrowser?

Reply:
It could use a method like this, what's your opinion?
~~~
historyBrowserClass
	"Answer the Presenter class to instantiate for browsing the history of the repository.
	The choices are IceTipHistoryBrowser or IceTipHiedraHistoryBrowser (the case if Hiedra v2.0.0 is loaded)."

	^ (self class environment hasClassNamed: #HiColumnController)
		ifTrue: [ IceTipHiedraHistoryBrowser ]
		ifFalse: [ IceTipHistoryBrowser ]
~~~

Reply:
I mean, then the line should be:
~~~
historyPanel := self instantiate: self historyBrowserClass on: self model headModel.
~~~

Reply:
Note: I pressed on "start review" by mistake.